### ISSUE TITLE
What Team Members are working on. 

### ISSUE THREAD

**Issue by Madonahs** [Madonahs/GrowWithGoogleTeamProject#35]
Hello ; Since i Know every team member is busy and it is hard to post on slack or Trello all the time, I have decided to create this issue with briefs. It will be updated regularly  so team members can be checking here. 

Madonah Syombua - I am working on Drawer Navigation functions | Projects . Branch - Interest-Activity

Rado - Is working on Google Login - Branch Google -Login

Ayo - is working on Drawer Navigation settings - Theme color changes

Hement is working on - Feeds Fragment | Interest Branch - Interest-Activity

Chuk is working on Profile Activity - Branch profile-fragment

Mahar is working on FireBase - i believe he is done we can all look at it. branch firebase -login 

Please Feel free to let me know where i can advance. Or should i create separate issues for everyone? 

Thank you team for being the BEST!!! 

On the Nav Drawer please let me know what you think, you can checkout my work on my branch.

**Comment by Madonahs** [comment]
Can we Applaud @MaherSoua  Login is now successful.  Thank you team cc @dri94  Well done Guys. 👍 💯

**Comment by emmaunel** [comment]
By the way guys, I created a slack workspace for us. The link is [growwithgoogleteam.slack.com](https://growwithgoogleteam.slack.com/). Please join so our communication can be better

**Comment by Madonahs** [comment]
@emmaunel  Thanks, Oh yeah we can create tabs for all work needed and each member can brief the other there. Nice idea. Easier too.

**Comment by MaherSoua** [comment]
It is better to assign each task to the right person and each issues will evolved alone, better than putting all issues in one issues and update it each time, it will be hard to follow many issues like that by the time. 
I will create them now

**Comment by MaherSoua** [comment]
Another thing that is not clear that @Madonahs and @HemantNimje work on the same branch, is there any reason to do so?
